HW2- SQL File

```diff
@@ -0,0 +1,78 @@
+/*-- BDA 696 HW 2 --
+ * Karenina Zaballa
+
+
+INSTRUCTIONS:
+
+GOAL: Calculate the batting average using SQL queries for every player
+
+ 1. Start with one player
+ 		Do Annual
+ 		Do Historic
+ 		Rolling (over last 100 days) ? - Look at the last 100 days that player was in prior to this game
+ 2. Apply above to all players
+   Store all these results in their own tables
+*/
+
+show errors;
+commit;
+
+USE baseball_db
```

Comment:
Not sure if you want to call out that your file is called "baseball_db" but it may be different depending on the user?

Reply:
Good catch, but not the end of the world. I went ahead and gave everyone in their class their own DB (so everyone had a fresh one) and I just changed this line everytime.